packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -14,6 +14,20 @@
 }
 
 :local {
+
+  .site-alert {
+    position: fixed;
+    background: var(--terra-consumer-site-alert-background, #000);
+    color: var(--terra-consumer-site-alert-color, #FFF);
+    bottom: 0;
+    width: 100%;
+    left: 0;
+    height: var(--terra-consumer-site-alert-height, 30px);
+    text-align: center;
+    line-height: var(--terra-consumer-site-alert-height, 30px);
```

Comment:
If the line height and height aren't the same it won't be vertically centered anymore right? Is there a good way to handle custom heights with always guaranteeing centered text?